app/helpers/layout_helper.rb

```diff
@@ -106,6 +106,7 @@ def multiple_checkboxes(f, attr, klass, associations, options = {}, html_options
   def multiple_selects(f, attr, associations, selected_ids, options = {}, html_options = {})
     options.merge!(:size => "col-md-10")
     authorized = AssociationAuthorizer.authorized_associations(associations).all
+    html_options["class"] = html_options["class"].nil? ? "without_select2" : html_options["class"] += " without_select2"
```

Comment:
This is a bit difficult to read. What do you think about changing the default for `html_options` above to `Hash.new('')`? 

That would allow you to reduce this to `html_options['class'] += ' without_select2 '`, as the default for non-found keys in the hash will be an empty string.